upb/def.c

```diff
@@ -1159,8 +1159,13 @@ const upb_ServiceDef* upb_DefPool_FindServiceByName(const upb_DefPool* s,
   return symtab_lookup(s, name, UPB_DEFTYPE_SERVICE);
 }
 
-const upb_FileDef* upb_DefPool_FindFileByNameforsym(const upb_DefPool* s,
-                                                    const char* name) {
+const upb_ServiceDef* upb_DefPool_FindServiceByNameWithSize(
+    const upb_DefPool* s, const char* name, size_t size) {
+  return symtab_lookup2(s, name, size, UPB_DEFTYPE_SERVICE);
```

Comment:
seems like we should have renamed symtab_lookup2 at some point...

Reply:
So far I've mainly focused on renames of public APIs, rather than static internal functions like `symtab_lookup2`. But I agree these are worth updating too when we get a spare moment.